docs/sources/reference/commandline/cli.md

```diff
@@ -698,7 +698,11 @@ to any of the files in the context. For example, your build can use an
 [*ADD*](/reference/builder/#add) instruction to reference a file in the
 context.
 
-The `URL` parameter can specify the location of a Git repository; the
```

Comment:
@jamtur01 I didn't add a list here because it felt like too much structure, since the three items reappear immediately after this paragraph as subsection headers (a kind of list). This sentence is supposed to be just a short heads up for the details that follow (does that make sense?)